core/code/hooks.js

```diff
@@ -74,7 +74,7 @@ window.runHooks = function(event, data) {
         return false;  //break from $.each
       }
     } catch (e) {
-      log.error('error running hook '+event+', error: '+e);
+      log.error('error running hook', { event: event, error: e, source: callback.toString(), data: data });
```

Comment:
Have you tried that in IITCm?
I suspect that it is unable to show objects in console.

Reply:
It will work in IITCm. It won't work in IITCm console window.